### PR TITLE
Add: Joke segment in `config-iparam-html-freshdesk` sample app

### PR DESCRIPTION
**Pull request by teja-kummarikuntla**
- Add: Card with question and punchline at `full_page_app`
Screenshot 2021-05-01 at 8 40 12 PM
Screenshot 2021-05-01 at 8 40 25 PM

### CODE REVIEW

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparam-html-freshdesk/app/styles/generics.css [hunk 1]
```diff
@@ -16,3 +16,37 @@ main {
   font-size: 1rem;
   font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
```
Comment (Saif-Shines) [inline Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparam-html-freshdesk/app/styles/generics.css:17]: Let's keep it simple. Let's not have more than 2 fallback font names.

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparam-html-freshdesk/app/styles/generics.css [hunk 2]
```diff
@@ -16,3 +16,37 @@ main {
...
+  -ms-flex-direction: column;
+  flex-direction: column;
+  min-width: 0;
+  word-wrap: break-word;
+  background-color: #fff;
+  background-clip: border-box;
+  border: 1px solid rgba(0, 0, 0, .125);
+  border-radius: .25rem;
+  padding: 10px; 
+  margin: 100px;
```
Comment (Saif-Shines) [inline Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparam-html-freshdesk/app/styles/generics.css:36]: Developers come to this repository to see the source code. Is it possible to optimize CSS rules in way simpler terms? For example, I assume `-webkit-*` is generally used to extend support to Safari browsers. Please feel free to assume most browsers are run the V8 JS engine and Blink rendering engine. So special rules are maybe redundant.

File: Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparam-html-freshdesk/app/styles/generics.css [hunk 3]
```diff
@@ -16,3 +16,37 @@ main {
...
+    flex: 1 1 auto;
+    padding: 1.25rem
+}
+
+.card-title {
+  margin-bottom: .75rem
+}
+
+.text-center {
+  text-align: center !important
```
Comment (Saif-Shines) [inline Freshworks-Samples/App-Development-Features/Configuration-Features/config-iparam-html-freshdesk/app/styles/generics.css:51]: Is there a possibility to not use `!important`  ?

**Review by ravirajsubramanian — APPROVED** [review]

**Review by Saif-Shines — APPROVED** [review]
Nicely covered

**Pull request merged**